sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala

```diff
@@ -881,6 +882,28 @@ class FileBasedDataSourceSuite extends QueryTest
       }
     }
   }
+
+  test("SPARK-32827: Add spark.sql.maxMetadataStringLength config") {
+    withTempDir { dir =>
+      val tableName = "t1"
+      val path = s"${dir.getCanonicalPath}/$tableName"
+      withTable(tableName) {
+        sql(s"create table t1(c int) using parquet location '$path'")
```

Comment:
plz use uppercases for SQL keywords, e.g., `CREATE TABLE`

Reply:
done.